mindmeld/components/entity_resolver.py

```diff
@@ -43,6 +49,14 @@
     resolve_es_config_for_version,
 )
 
+try:
+    from sentence_transformers import SentenceTransformer
+    from sentence_transformers.models import Transformer, Pooling
+
+    sbert_available = True
+except ImportError:
+    sbert_available = False
```

Comment:
I think these optional requirements should on imported dynamically using `importlib.import_module`